src/app/core/index/index.selectors.ts

```diff
@@ -3,6 +3,7 @@ import { hasValue, isNotEmpty } from '../../shared/empty.util';
 import { coreSelector } from '../core.selectors';
 import { URLCombiner } from '../url-combiner/url-combiner';
 import { IndexState, MetaIndexState } from './index.reducer';
+// eslint-disable-next-line import/no-namespace
 import * as parse from 'url-parse';
```

Comment:
This import can be left out, we can replace
* `parse(...)` → `new URL(...)`
* `parsed.query` → `parsed.search`